calico_containers/pycalico/netns.py

```diff
@@ -209,6 +251,18 @@ def remove_ip_from_ns_veth(namespace, ip, veth_name_ns):
                        "dev", veth_name_ns])
 
 
+class Route(object):
+    def __init__(self, route_output):
+        self.route_output = route_output
+        self.default = route_output.startswith("default")
+        match = re.search('metric\s+(\d+)', route_output)
+        self.metric = int(match.group(1)) if match else 0
+
+    def __str__(self):
+        route_without_metric = re.sub('metric\s+\d+', '', self.route_output)
```

Comment:
I opted for simplicity by always specifiying the metric even if its 0, but I could do this:
```
route_without_metric = re.sub('metric\s+\d+', '', self.route_output)
if self.metric:
    return "{} metric {}".format(route_without_metric, self.metric)
else:
    return route_without_metric
```
I figure being explicit takes up less lines, so I just always print the metric. Up for suggestions though.